NEWS

```diff
@@ -1,3 +1,85 @@
+0.5.19 -- 23 January 2015.
+
+Support class conversion such as as.Date() within ggplot code.
+
+0.5.18 -- 22 January 2015.
+
+Return proper filepath when filename contains directories.
+
+0.5.17 -- 30 December 2014.
+
+Support date-time binning in histograms.
+
+0.5.16 -- 29 December 2014.
+
+Support colour aesthetic in geom_text().
+
+0.5.15 -- 19 December 2014.
+
+Use proper RCurlOptions in get_figure() method.
+
+0.5.14 -- 1 December 2014.
+
+Make layers geom_line + geom_point only one trace in Plotly.
```

Comment:
I re-implemented this functionality in toby-fixes branch, at the end of gg2list.